packages/manager/modules/nutanix/src/dashboard/component/add-nodes-modal/controller.js

```diff
@@ -2,9 +2,10 @@ import { MAX_NODES_BY_CLUSTER } from '../../../constants';
 
 export default class {
   /* @ngInject */
-  constructor($window, $translate) {
+  constructor($window, $translate, $locale) {
     this.$translate = $translate;
     this.$window = $window;
+    this.$locale = $locale;
```

Comment:
Could be computed only once

```suggestion
    this.userLanguage = $locale.localeID.replace('_', '-');;
```